test/unit/contextResponses/blackButtonPollingRequestQuerySuccess.json

```diff
@@ -13,7 +13,10 @@
           },
           {
             "name": "op_result",
-            "value": "999999",
+            "value": {
+              "code": 200,
```

Comment:
I saw you used string codes in 'blackButtonSynchLazyRequest.json' and 'blackButtonSynchLazyRequestCodified.json' files but a number (integer) in this case... :) I guess your code is smart enough to deal with both types, right? :) In fact, this made me rethink if the CA for the sake of flexibility (in the future mostly) should send a string as the code too... .:) What do you think?